pyzebra/load_1D.py

```diff
@@ -0,0 +1,181 @@
+import os
+import re
+from collections import defaultdict
+from decimal import Decimal
+
+import numpy as np
+
+META_VARS_STR = (
+    "instrument",
+    "title",
+    "sample",
+    "user",
+    "ProposalID",
+    "original_filename",
+    "date",
+    "zebra_mode",
+    "proposal",
+    "proposal_user",
+    "proposal_title",
+    "proposal_email",
+)
+META_VARS_FLOAT = (
+    "mf",
+    "2-theta",
+    "chi",
+    "phi",
+    "nu",
+    "temp",
+    "wavelenght",
+    "a",
+    "b",
+    "c",
+    "alpha",
+    "beta",
+    "gamma",
+    "cex1",
+    "cex2",
+    "mexz",
+    "moml",
+    "mcvl",
+    "momu",
+    "mcvu",
+    "detectorDistance",

```

Comment:
the detectorDistance is a number in ccl files, but in dat files its date, which breaks the code. I asked Oksana if it can be fixed in the software that produces the *.dat files. If it for whatever reason cant be fixed, I suggest to move it to META_VARS_STR.